packages/web/src/components/user-list/components/UserList.tsx

```diff
@@ -57,16 +60,18 @@ const UserList = (props: UserListProps) => {
               className={styles.artistChipContainer}
               popoverMount={MountPlacement.BODY}
             />
-            {user.user_id !== props.userId ? (
-              <FollowButton
-                size='small'
-                following={user.does_current_user_follow}
-                onFollow={() => props.onFollow(user.user_id)}
-                onUnfollow={() => props.onUnfollow(user.user_id)}
-                showIcon
-                stopPropagation
-              />
-            ) : null}
+            {user.user_id !== props.userId
+              ? props.renderActionButton?.(user.user_id) || (
```

Comment:
Unfortunately, this approach doesn't change what the onclick does for the whole row (well, the user chip part of the row). That might need revisiting...